spec/services/food_service_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Food API' do
+  it 'returns food data' do
+    food_data = FoodService.get_details("sweet potatoes")
+
+    expect(food_data).to be_an(Array)
+    expect(food_data.first).to be_a(Hash)
```

Comment:
This is the proper place to test the structure of your API response and the data types. These tests can then be removed from the facade test, and instead we should test a pure input/output that returns the exact value we expect.